sbin/build.sh

```diff
@@ -518,13 +518,13 @@ makeACopyOfLibFreeFontForMacOSX() {
 }
 
 
-# Get the first tag from the git repo
+# Get the tags from the git repo and choose the latest tag when there is more than one for the same SHA.
 # Excluding "openj9" tag names as they have other ones for milestones etc. that get in the way
 getFirstTagFromOpenJDKGitRepo()
 {
     git fetch --tags "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}"
     revList=$(git rev-list --tags --topo-order --max-count=$GIT_TAGS_TO_SEARCH)
-    firstMatchingNameFromRepo=$(git describe --tags $revList | grep jdk | grep -v openj9 | head -1)
+    firstMatchingNameFromRepo=$(git tag --points-at $revList | grep jdk | grep -v openj9 | tail -1)
```

Comment:
This probably wants to use `sort -V` like [1] to be sure to choose the highest version tag.

[1] https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/openj9/closed/autoconf/custom-hook.m4#L320

Reply:
As per my comment on an earlier version of this, `sort -V` is a GNU specific option so I don't really want to be using it (won't work with the default AIX sort)

Reply:
I hadn't noticed your earlier comment: sorry for that.
On the other hand, it _is_ working on all platforms where openj9 is built, including AIX.

Reply:
I suspect your openj9 systems have the GNU coreutils installed and in the PATH before the system ones.